omod/src/main/webapp/fragments/sendPatientToQueue.gsp

```diff
@@ -0,0 +1,83 @@
+<%
+    ui.includeJavascript("patientqueueing", "patientqueue.js")
+%>
+
+<script>
+    if (jQuery) {
+        jq(document).ready(function () {
+            jq("#create").click(function () {
+                jq.post('${ ui.actionLink("create") }', {
+                    patientId: jq("#patient_id").val().trim(),
+                    providerId: jq("#provider_id").val().trim(),
+                    locationId: jq("#location_id").val().trim()
+                }, function (response) {
+                    var jsonToastMessage = JSON.parse(response.replace("toastMessage=", "\"toastMessage\":").trim());
```

Comment:
What is the reason for the above replacement?

Reply:
@dkayiwa the format which the method returns is not purely json